dev_guide/builds.adoc

```diff
@@ -856,28 +856,37 @@ $ oc secrets new mysecret ca.crt=path/to/certificate
 $ oc secrets new mysecret ca.crt=path/to/certificate .gitconfig=path/to/.gitconfig
 ----
 ====
-
++
 [NOTE]
 ====
-Please note that SSL verification can be turned off, if `sslVerify=false` is set
-for the `http` section in your `.gitconfig` file:
+SSL verification can be turned off, if `sslVerify=false` is set for the `http`
+section in your `.gitconfig` file:
 ----
 [http]
         sslVerify=false
 ----
 ====
 
-. Add the `*secret*` to the builder service account. Each build is run with
+.  Add the `*secret*` to the builder service account. Each build is run with the
 `serviceaccount/builder` role, so you need to give it access your secret with
 following command:
-
 +
 ====
 ----
 $ oc secrets add serviceaccount/builder secrets/mysecret
 ----
 ====
 
+.  Add the `*secret*` to the `*BuildConfig*`:
+
+====
+[source,yaml]
+----
+type: "GitHub"
```

Comment:
Is this correct setting? I thought that we need to set it like this:

https://docs.openshift.org/latest/dev_guide/builds.html#using-secrets-in-the-buildconfig

~~~
source:
  git:
    uri: https://github.com/openshift/nodejs-ex.git
  secrets:
    - secret:
        name: "mysecret"
  type: Git
~~~

I am wondering that `"type": "GitHub"` is for webhook (Build Triggers).